api/api.yaml

```diff
@@ -2835,6 +2848,75 @@ components:
           description: The time the connection was last updated.
           format: date-time
           example: 2023-07-13T21:34:44.816354Z
+        authScheme:
+          type: string
+          description: The authentication scheme used for this connection.
+          example: oauth2
+          enum:
+            [
+              none,
+              apiKey,
+              basic,
+              oauth2/authorizationCode,
+              oauth2/authorizationCodePKCE,
+              oauth2/clientCredentials,
+              oauth2/password,
+              jwt,
+            ]
+        scopes:
+          type: array
+          description: The OAuth scopes for this connection.
+          items:
+            type: string
+        status:
+          type: string
+          description: The status of the connection.
+          example: active
```

Comment:
this example is not from the enum list, so either the example or the enum list is inaccurate.